frontend/lindsay-wildlife-treasure-hunt-frontend/src/Components/GameDisplay/GameDisplay.js

```diff
@@ -9,17 +9,22 @@ import ManualEntryDisplay from './ManualEntryDisplay';
 import TestZoomComponent from './TestZoomComponent';
 
 import './GameDisplay.css';
+import { ReedSolomonException } from '@zxing/library';
```

Comment:
Do we need this?

Reply:
No we don't need this. This got automatically added to the imports after my IDE tried to autocorrect something else I wrote into a ReedSolomanException, whatever that is. Good catch, I missed it.